### PR TITLE
Deprecate direct use of CoreView

### PR DESCRIPTION
**Pull request by wagenet**
I'm not 100% sure this is worth it since we do plan to deprecate View altogether.

### CODE REVIEW

File: packages/ember-views/lib/views/core_view.js [hunk 1]
```diff
@@ -153,4 +154,12 @@ CoreView.reopenClass({
   isViewClass: true
 });
 
+export var DeprecatedCoreView = CoreView.extend({
+  init: function() {
+    console.log('init');
```
Comment (rwjblue) [inline packages/ember-views/lib/views/core_view.js]: Extra console.log here

**Comment by stefanpenner** [comment]
+1 other then test failure + what @rwjblue mentioned

**Comment by wagenet** [comment]
Fixed the console.log.

**Comment by wagenet** [comment]
@rwjblue Should this have a deprecation link?

**Pull request merged**